cloudwatch-logs-s3-forwarder/example/default/main.tf

```diff
@@ -0,0 +1,22 @@
+# The key of the map is the cloudwatch log group, the value is the filter syntax expression.
+# All matching log entries will be forwarded to a Lambda function, that forwards the entries to
+# a shared bucket in the telia-common-logs-prod account.
+
+locals {
+  subscriptions = {
+    "test" = ""
+  }
+}
+
+module "cloudwatch_splunk_lambda_subscription" {
+  source                 = "../.."
+  lambda_artifact_s3_key = "cloudwatch-logs-remote-bucket-1.1.zip"
+  log_group_names        = "${keys(local.subscriptions)}"
+  filter_patterns        = "${values(local.subscriptions)}"
+  lambda_name            = "telia-log-forwarder"
+  lambda_bucket          = "telia-common-logs-prod-lambda"
+  log_bucket_name        = "telia-common-logs-prod-application-logs"
```

Comment:
Are these "real" values? If so we should probably change them to dummy values.

Reply:
Will do.